html-formatter/javascript/package.json

```diff
@@ -21,18 +21,15 @@
     "build": "tsc && webpack-cli && cp src/index.mustache.html dist/src"
   },
   "dependencies": {
```

Comment:
The ones left in `dependencies` are those used by the CLI and/or the CucumberHtmlStream.